static/css/devhub/base.less

```diff
@@ -0,0 +1,97 @@
+@import '../impala/fonts/fira/fira.less';
+
+// Colors
+@primary-font: 'Fira Sans', sans-serif;;
+
+// Common styles
+section {
+  padding: 0 20px 60px 20px;
+
+  & > h1 {
+    font-size: 40px;
+    line-height: 1.2;
+    margin-bottom: 0;
+    margin-top: 20px;
+  }
+
+  & > p {
```

Comment:
If this a style for a common paragraph and heading, does it need to be nested? It's also seems like it might be a bit too specific being a direct descendent selector.